WalletWasabi.Tests/UnitTests/WabiSabi/CredentialTests.cs

```diff
@@ -111,14 +104,14 @@ public void CredentialIssuance()
 
 				var credentialResponse = issuer.HandleRequest(credentialRequest);
 				client.HandleResponse(credentialResponse, validationData);
-				Assert.Equal(ProtocolConstants.CredentialNumber, client.Credentials.ZeroValue.Count());
-				Assert.Empty(client.Credentials.Valuable);
-				var issuedCredential = client.Credentials.ZeroValue.First();
+//				Assert.Equal(numberOfCredentials, client.Credentials.ZeroValue.Count());
+//				Assert.Empty(client.Credentials.Valuable);
```

Comment:
Same.

Reply:
The first one should be remove. The second one makes sense.

Reply:
Done.